### ISSUE TITLE
Node-Red Automatons stop working after some time

### ISSUE THREAD

**Issue by DiddyDaveUK** [hassio-addons/addon-node-red#424]
# Problem
Node-Red automatons stop working after an unknown time as it is random.  Once the service restarted the automatons then start working again until the next time.

It's roughly about 4 to 6 hours before the automatons stop working.  

My current workaround is to do a yaml automation to restart the hassio addon every 4 hours.

I'm using the 
Hassio 103.6
Node-RED Addon 6.0.1

**Comment by sinclairpaul** [comment]
It's a little difficult to help here, could you share some more information:

Is it Homeassistant based flows only? Or any flow in Node Red?
Is anything logged in the addon log, or the Supervisor log?
How is Node Red configured for Hassio?

**Comment by DiddyDaveUK** [comment]
Yes they are all Home Assistant automatons.
There is no errors in the logs that I can see but will keep an eye out
The Node-RED is installed via the Hassio Store and the Home Assistant integration was installed via HACS

**Comment by mr-awk** [comment]
I'm experiencing that some automations in nodered is not work as expected, especially with lightschedule module. Saw now that there is an update to that module.

**Comment by DiddyDaveUK** [comment]
I think I have found out what my issue was.  Since doing this my Node-RED has been working fine without and restarts.
In the settings below I created a user and pass for both of these as I did think I needed them.  As part of my investigation I found that if I left these as below and just had the main password in place

"credential_secret": "Just this password In Place",

"http_node": {
    "username": "", ( Leave these as is )
    "password": "" ( Leave these as is )
  },
  "http_static": {
    "username": "", ( Leave these as is )
    "password": "" ( Leave these as is )

With the above settings My Node-RED is working as it should.  I'm not saying that this would be the fix for everyone but it is for me.  I'm also not sure if this is safe either but as my system is local only I think it should be fine.

Thanks
Dave
